composer.json

```diff
@@ -23,8 +23,7 @@
     "require": {
         "php": "^7.4 || ^8.0",
         "cocur/slugify": "^3.0 || ^4.0",
-        "doctrine/doctrine-bundle": "^1.12.3 || ^2.0",
-        "doctrine/persistence": "^1.3.4 || ^2.0",
```

Comment:
you should not remove this requirement. It is used on the code. Maybe you want to drop lower version?

Here you have the requirements for 4.x branch:

```
        "doctrine/doctrine-bundle": "^2.5",
        "doctrine/persistence": "^2.1",
```